provision/acc_provision/acc_provision.py

```diff
@@ -2127,8 +2115,21 @@ def provision(args, apic_file, no_random):
     config["aci_config"]["sync_login"]["cert_data"] = cert_data
     config["aci_config"]["sync_login"]["cert_reused"] = reused
 
+    if not is_calico_flavor(config["flavor"]):
+        global ACI_CNI_VERSIONS_PATH
```

Comment:
Use of global is generally discouraged. One option would have been to set these paths as key/values in the "config" dict. (in case we want to use these paths even outside this module). If it seems like an overkill, we can stay with current pattern.

Reply:
Ohkk Sumit, Let me quickly have it, should not be a big change

Reply:
we can skip this or take it as follow up (but I would say we should do the other change for cko + netop_version)